chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/developer/attestation/email/missingchangerequest/MissingAttestationChangeRequestDeveloperStatusReportEmailGenerator.java

```diff
@@ -53,10 +53,14 @@ private String getMessage(List<DeveloperEmail> developerEmails, UserDTO submitte
     }
 
     private List<List<String>> getTableRows(List<DeveloperEmail> developerEmails) {
+        if (developerEmails.size() > 0) {
         return developerEmails.stream()
```

Comment:
Is this indentation correct? This `return` block was encapsulated in an `if` block, so it should need to change indentation, right?